BBBSetup/initGPIO

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+echo 37 > /sys/class/gpio/export
```

Comment:
This is exactly right.  I haven't tested it, but this exactly the script that is needed